src/app/locale/masterpass-supported-locales.ts

```diff
@@ -0,0 +1,15 @@
+export const supportedLocales: {[language: string]: string[]} = {es: ['es_es', 'es_mx', 'es_pe', 'es_co', 'es_ar', 'es_cl'],
```

Comment:
Why are we validating the locales against this list rather than just formatting it and letting the Masterpass domain decide what to do?

The domain seems to have logic to handle non supported locales:

https://sandbox.src.mastercard.com/assets/img/btn/src_chk_btn_126x030px.svg?paymentmethod=master,visa,amex,discover&locale=xx_xx

Whereas we seem to be trying to force a defaulting to 'en_us'

Reply:
We validate the locales based on the masterpass supported locale list so it always shows the new button.
We use getLanguageService().getLocale() to fetch the locale, but this gives back a language only locale like 'en', so we need to validate it against a list to get a language_country locale.
I can change this to get the locale from the store profile, which gives back a language_country locale

Reply:
I wasn't suggest that we change the source of the locale, more questioning the need to filter it.

Do we want the button's display based on the store's locale or the customer's locale?

If we _do_ want the customers I think we'll need to read it from `window.navigator.language`, though I'm not sure about browser support or formatting gotchas, so maybe look into that if it is needed.

Reply:
We want to show it based on the store locale